### PR TITLE
升级最新依赖 并修复eslint报错

### PR DESCRIPTION
**Pull request by xingyuv**
### `General`

> ✏️ Mark the necessary items without changing the structure of the PR template.

- [ ] Pull request template structure not broken

### `Type`

> ℹ️ What types of changes does your code introduce?

> 👉 _Put an `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### `Checklist`

> ℹ️ Check all checkboxes - this will indicate that you have done everything in accordance with the rules in [CONTRIBUTING](contributing.md).

> 👉 _Put an `x` in the boxes that apply._

- [ ] My code follows the style guidelines of this project
- [ ] Is the code format correct
- [ ] Is the git submission information standard?
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by kirklin** [comment]
vue升级3.3.4的话，unplugin-vue-define-options可以移除掉了

File: internal/ts-config/package.json [hunk 1]
```diff
@@ -19,8 +19,8 @@
     "node-server.json"
   ],
   "dependencies": {
-    "@types/node": "^18.15.11",
-    "unplugin-vue-define-options": "^1.3.3",
-    "vite": "^4.3.0-beta.2"
+    "@types/node": "^20.3.3",
+    "unplugin-vue-define-options": "^1.3.8",
```
Comment (kirklin) [inline internal/ts-config/package.json:23]: vue3.3.4已经支持了

File: README.zh-CN.md [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
 ## 简介
 
-Vue Vben Admin 是一个免费开源的中后台模版。使用了最新的`vue3`,`vite2`,`TypeScript`等主流技术开发，开箱即用的中后台前端解决方案，也可用于学习参考。
+Vue Vben Admin 是一个免费开源的中后台模版。使用了最新的`vue3`,`vite4`,`TypeScript`等主流技术开发，开箱即用的中后台前端解决方案，也可用于学习参考。
```
Comment (kirklin) [inline README.zh-CN.md:12]: vite建议可以不用指定具体版本

**Review by kirklin — CHANGES_REQUESTED** [review]

**Comment by scottMan1001** [comment]
能解决这个问题吗https://github.com/vbenjs/vue-vben-admin/issues/2898

File: internal/ts-config/package.json (hunk 1 above)
Comment (xingyuv) [reply internal/ts-config/package.json]: done

**Comment by xingyuv** [comment]
> 能解决这个问题吗#2898

解决了

**Comment by scottMan1001** [comment]
什么时候能合入呢？

**Comment by fzxiang** [comment]
这条pr 依赖升级会引发的严重问题， 复现： clone新仓库，依赖安装完后执行lint 指令，会报如下错误
image
stylelint 傻掉了 找不到命令。

**Pull request merged**